src/main/java/module-info.java

```diff
@@ -1,3 +1,4 @@
 module cafe.cryptography.ed25519_elisabeth {
     requires cafe.cryptography.curve25519_elisabeth;
+    requires annotations;
```

Comment:
```suggestion
    requires org.jetbrain.annotations_java5;
```
(see [here](https://github.com/JetBrains/java-annotations/blob/master/build.gradle#L43))